mlrun/frameworks/sklearn/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""
+Description:
+__init__ function of sklearn-autologger. Will be extended and contain multiple Sklearn-specific functions.
+"""
+
+import mlrun
+from mlrun.frameworks.mlbase.mlrun_interface import MLBaseMLRunInterface
+from mlrun.frameworks._common.pkl_model_server import PklModelServer
+
+SklearnModelServer = PklModelServer
```

Comment:
add comment `# placeholder, SklearnModelServer may deviate from PklModelServer in future ver`